pages/companies/[slug]/index.tsx

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+// import { useRouteMatch } from 'react-router-dom';
+import SlugContainer from '../../../components/slug/SlugContainer';
+import CompanyHeader from '../../../components/slug/CompanyHeader';
+import CompanyProfile from '../../../components/slug/CompanyProfile';
+import CompanyContacts from '../../../components/slug/CompanyContacts';
+import QuantityAdminsPartners from '../../../components/slug/QuantityAdminsPartners';
+import TableRender from '../../../components/slug/TableRender';
+import PreviousInfo from '../../../components/slug/CompanyPreviousInfo';
+import CompanyInvestCapital from '../../../components/slug/CompanyInvestCapital';
+import SimilarCompanies from '../../../components/slug/SimilarCompanies';
+import CompanyTurnover from '../../../components/slug/CompanyTurnover';
+import { ISlug } from '../../../types/slug';
+import styles from '../../../styles/Home.module.css';
+import Loading from '../../../components/Loading';
+
+axios.defaults.baseURL = 'https://app.informer.md/api/public';
```

Comment:
Add baseUrl in `.env`, also add an `.env.example`, like:
NEXT_PUBLIC_URL=https://app.informer.md/api/public

It is also best to initialize axios separately and after using it through import, like:
`libs/axios.ts`:
```javascript
import $axios, { CancelToken } from 'axios';

export const axios = $axios.create();

axios.defaults.headers.post['Content-Type'] = 'application/json';
axios.defaults.baseURL = process.env.NEXT_PUBLIC_URL;

// It is like a middle-ware
axios.interceptors.response.use(
  (response) => response,
  (error) => {
    if ($axios.isCancel(error)) {
      return Promise.reject({ cancel: true, message: 'The endpoint was cancelled' });
    }

    return Promise.reject(error);
  },
);

// Update accept language
export const axiosHeadersUpdater = (lang: string) => {
  axios.defaults.headers['Accept-Language'] = lang;
};

// Set default accept language
axiosHeadersUpdater(Locale.RO);
```